### ISSUE TITLE
Move Documentation under About?

### ISSUE THREAD

**Issue by christinabergmann** [langcog/metalab#6]
I think a lot of what we have under documentation (statistical approach etc) can also live under about. 

Objections?

**Comment by lottiegasp** [comment]
I think the tab on the homepage currently called "Documentation" should be changed to "About".

Then under "About" (currently "Documentation"), the subheading currently called "Applications" is changed to "Application documentation". As it took me awhile to realise that the tools themselves don't live here, but are under "Explore Data"

Was it a conscious decision to put the tutorials under the "Documentation" tab? I would suggest there is an additional tab "Tutorials" along with "Explore Data" and "About" (currently "Documentation"). which we can discuss the structure of, along the lines of the suggestions here https://github.com/langcog/metalab/issues/7 and here https://docs.google.com/document/d/16RDwrsTF9_5ku73CvMplOwdYyLL8uFHzH21Ysfv3r6Q/edit

**Comment by christinabergmann** [comment]
I don't think it was a conscious decision, it just sort of grew and it could be structured more intuitively. This is all very much in flux, so let's try to optimize it for the naive user!
